packages/phoenix-ng/projects/phoenix-app/cypress/e2e/user-scenarios.test.ts

```diff
@@ -0,0 +1,76 @@
+describe('User scenarios', () => {
+  it('should generate a shareable link based on specified params: JiveXML', () => {
+    cy.visit('/trackml');
+    cy.get('app-share-link').click();
+
+    cy.get('#eventDataFile').type('test_path');
+    cy.contains('test_path').should('be.visible');
+    cy.get('#eventDataType').select('JiveXML');
+    cy.get('#eventDataType').should('have.value', 'jivexml');
+    cy.get('#configuration').type('test_path2');
+    cy.contains('test_path2').should('be.visible');
+
+    cy.get('.share-box')
+      .eq(0)
+      .contains(
+        Cypress.config().baseUrl +
+          'trackml?file=test_path&type=jivexml&config=test_path2'
+      )
+      .should('be.visible');
+    cy.get('.share-box')
+      .eq(1)
+      .contains(
+        '<iframe src="' +
+          Cypress.config().baseUrl +
+          'trackml?embed=true&file=test_path&type=jivexml&config=test_path2"></iframe>'
+      )
+      .should('be.visible');
+
+    cy.contains('Close').click();
+  });
+
+  it('should generate a shareable link based on specified params: JSON', () => {
+    cy.visit('/trackml');
+    cy.get('app-share-link').click();
+
+    cy.get('#eventDataFile').type('test');
+    cy.contains('test').should('be.visible');
+    cy.get('#eventDataType').select('JSON');
+    cy.get('#eventDataType').should('have.value', 'json');
+    cy.get('#configuration').type('test2');
+    cy.contains('test2').should('be.visible');
+
+    cy.get('.share-box')
+      .eq(0)
+      .contains(
+        Cypress.config().baseUrl + 'trackml?file=test&type=json&config=test2'
+      )
+      .should('be.visible');
+    cy.get('.share-box')
+      .eq(1)
+      .contains(
+        '<iframe src="' +
+          Cypress.config().baseUrl +
+          'trackml?embed=true&file=test&type=json&config=test2"></iframe>'
+      )
+      .should('be.visible');
+
+    cy.contains('Close').click();
+  });
+
+  it('should rotate geometry', () => {
```

Comment:
Does this produce the same result across multiple runs? If it doesn't, I think it's fine to remove it. It's a continuous animation and would be hard to consistently get the same image snapshot.